### DISCUSSION TITLE
Docker error

### DISCUSSION THREAD

**Discussion by Skeferstat** [Netflix/conductor#3307] [Q&A] [answered]
Hi, I want starting the docker container, but get following error:

[conductor:server builder 4/4] RUN ./gradlew build -x test --stacktrace:

#0 0.508 /bin/sh: 1: ./gradlew: not found

failed to solve: executor failed running [/bin/sh -c ./gradlew build -x test --stacktrace]: exit code: 127

I cloned the current repository... What I am doing wrong?
Thanks for help!

**Comment by Skeferstat** [comment, marked as answer]
./gradle file should have LF format.

**Comment by Taqi-Techverx** [comment]
Having same issue

**Comment by manan164** [comment]
Hi @Taqi-Techverx  ,
Please follow the steps,
1. Delete your current conductor folder
2. Run **git config --global core.autocrlf false**; 
3. Run **git config --global core.eol lf**; (Small L and f)
4. Clone conductor repo again
5. Check if end of lines shows as CRLF via NotePad++ by opening a DockerFile, e.g. => conductor/docker/server/DockerFile
6. Run docker-compose up on conductor/docker

**Reply by Kiranbo97** [reply]
Hi sir 
after build where data is stored

conductor-redis-persistence module uses [Dynomite](https://github.com/Netflix/dynomite) for storing the state and metadata along with [Elasticsearch](https://www.elastic.co/) for indexing backend.

**Reply by Kiranbo97** [reply]
this conductor which data base is used to store the data and how we find the .properties please guid me sir

**Reply by manan164** [reply]
Hi @Kiranbo97 , Redis contains workflow execution information and Elasticsearch is used as persistence db. [Here](https://github.com/Netflix/conductor/blob/main/docker/server/config/config.properties) is the properties file. Let me know if this works, or we can chat [here](https://join.slack.com/t/orkes-conductor/shared_invite/zt-xyxqyseb-YZ3hwwAgHJH97bsrYRnSZg) for more realtime collaboration.

**Reply by Kiranbo97** [reply]
thank u so much sir

**Reply by VinayakDataScience** [reply]
Hi Manan,

I have tried this on the Windows machine, but  didn't work.
windows simple set it to true:

$ git config --global core.autocrlf true

 macOS set it to false:

$ git config --global core.autocrlf false

**Comment by deltoss** [comment]
Thanks, I was having the same issue. I did some tinkering after the issue was fixed with the above solution. Instead of turning `autocrlf` off and configure `eof` to be `lf` on user git configs, I'm thinking we could add:

```text
gradlew eol=lf
*.gradle eol=lf
*.java eol=lf
*.groovy eol=lf
spring.factories eol=lf
```

To `.gitattributes` to prevent this issue. Thoughts? Happy to raise a PR.
